src/sql-parser/src/parser.rs

```diff
@@ -1160,6 +1166,37 @@ impl<'a> Parser<'a> {
         }))
     }
 
+    /// Parse a possibly qualified, possibly quoted operator, e.g.
+    /// `+`, `pg_catalog.+`, or `"pg_catalog".+`
+    fn parse_operator_name(&mut self) -> Result<Op, ParserError> {
+        let mut schema = vec![];
+        let original_pos = self.index;
+        while !matches!(self.peek_token(), Some(Token::Op(_)) | Some(Token::Star)) {
+            schema.push(self.parse_identifier()?);
+            self.expect_token(&Token::Dot)?;
+        }
+
+        // currently, we only support pg_catalog
```

Comment:
This comment just restates the `if` condition—would be good to get more of the "why" into the comment!